src/Model/Order.php

```diff
@@ -202,4 +202,27 @@ private function log($content)
             $this->settings->log()->add($file, $message);
         }
     }
+
+    /**
+     * See if the order needs processing before it can be completed.
+     * @return bool
+     */
+    public function needs_processing() {
+        $needs_processing = false;
+        if ( count( $this->wc_order->get_items() ) > 0 ) {
```

Comment:
Manter somente duas tabulações...
sugestão:

if ( !count( $this->wc_order->get_items() )) {
    return false;
}